buffer/src/main/java/io/netty/buffer/CompositeByteBuf.java

```diff
@@ -648,6 +660,20 @@ protected int _getUnsignedMedium(int index) {
     }
 
     @Override
+    protected int _getUnsignedMediumLE(int index) {
+        Component c = findComponent(index);
+        if (index + 3 <= c.endOffset) {
+            return c.buf.getUnsignedMediumLE(index - c.offset);
+        } else if (order() == ByteOrder.BIG_ENDIAN) {
+            return _getShortLE(index) & 0xFFFF |
```

Comment:
@ifesdjeen - Yip I realized you were just following what was already there...but might as well make more consistent while we are here.  Thanks :smile: